runtime/rasdump/TextFileStream.cpp

```diff
@@ -68,7 +68,7 @@ void
 TextFileStream::close(void)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
-	if (_FileHandle != -1) {
+	if (_FileHandle != -1 && _FileHandle != OMRPORT_TTY_OUT && _FileHandle != OMRPORT_TTY_ERR) {
```

Comment:
The check for TTY_OUT/ERR needs to done around the close. The j9file_write_text() and j9file_sync() calls are still desired.